spec/spec_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'pry'
 require 'active_model'
-require 'graphiti-activegraph'
+require 'graphiti_activegraph'
```

Comment:
This should be `graphiti-activegraph`
https://rubygems.org/gems/graphiti-activegraph/versions/0.1.11